sway/main.c

```diff
@@ -272,6 +274,7 @@ int main(int argc, char **argv) {
 		"  -c, --config <config>  Specify a config file.\n"
 		"  -C, --validate         Check the validity of the config file, then exit.\n"
 		"  -d, --debug            Enables full logging, including debug information.\n"
+		"  -H, --headless         Starts sway with the headless backend.\n"
```

Comment:
If we do the "no backends at all until configured explicitly" thing this should probably be `--no-auto-backend`